charts/app/templates/appealsServiceApi.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "app.fullname" . }}-appeals-service-api
+  labels:
+    {{- include "app.labels" . | nindent 4 }}
+spec:
+{{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.appealsServiceApi.replicaCount }}
+{{- end }}
+  selector:
+    matchLabels:
+      {{- include "app.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+    {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      labels:
+        {{- include "app.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "app.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.appealsServiceApi.image.repository }}:{{ .Values.appealsServiceApi.image.tag }}"
+          imagePullPolicy: {{ .Values.appealsServiceApi.image.pullPolicy }}
+          ports:
+            - name: appeals-srv-api
+              containerPort: 3000
+              protocol: TCP
+          env:
+            - name: MONGODB_URL
+              valueFrom:
+                secretKeyRef:
+                  key: url
+                  name: {{ .Values.appealsServiceApi.secrets.mongodb.appeals }}
+            - name: SERVER_PORT
+              value: {{ 3000 | quote }}
```

Comment:
Is it possible to use the variable (https://github.com/foundry4/appeal-planning-decision/pull/59/files#diff-f9b76d9e4b03537e44f2a2103cb3a267a0bf92478b0d975bc974fe17e0cde7b6R9) here, and on line 38 also? 

(no idea if it **is** possible, just a Q and potential improvement if so?)

Reply:
Yeah, this is one of those things I was in 50/50 about. The fact you've asked the question has tipped the balance in favour of using a variable